frontend/src/helpers/user-service/api-wrapper.ts

```diff
@@ -0,0 +1,87 @@
+import axios from "axios";
+import { UserInfo } from "../../components/common/ReadOnlyUserCard";
+
+const api = axios.create({
+  baseURL:
+    process.env.USER_API_ENDPOINT ?? "http://localhost:3001/user",
+  timeout: 5000,
+  headers: {
+    "Content-type": "application/json",
+  },
+});
+
+const login = async (email: string, password: string): Promise<User> => {
+    const response = await api.post(
+        `/login`,
+        {
+            email: email,
+            password: password
+        },
+        { withCredentials: true}
+    ).then((res) => {
+        if (res.status == 200) {
```

Comment:
Please use `HttpStatusCode.OK` for consistency.

Reply:
Resolved